static/app/views/performance/onboarding.tsx

```diff
@@ -93,6 +100,25 @@ type Props = {
 function Onboarding({organization, project}: Props) {
   const api = useApi();
 
+  const {projects} = useProjects();
+  const {location} = useRouteContext();
+
+  const {projectsForOnboarding} = filterProjects(projects);
+
+  const showOnboardingChecklist = organization.features?.includes(
+    'performance-onboarding-checklist'
+  );
+
+  useEffect(() => {
+    if (
+      showOnboardingChecklist &&
+      location.hash === '#performance-sidequest' &&
```

Comment:
Do you still need this, given that we're already doing this in `static/app/components/sidebar/index.tsx`?